packages/weex-vue-framework/factory.js

```diff
@@ -2590,8 +2590,7 @@ function normalizeScopedSlot(normalSlots, key, fn) {
       ? [res] // single vnode
       : normalizeChildren(res);
     return res && (
-      res.length === 0 ||
-      (res.length === 1 && res[0].isComment) // #9658
+      res.every(r => r.isComment) // #9658, #11856
```

Comment:
It should still check that `res.length === 0`, no?

Reply:
`Array#every` does return true if its length is 0

Reply:
I think length check if zero, that is unnecessary